mbtools/validate_mbz_html.py

```diff
@@ -189,13 +193,40 @@ def check_duplicate_uuid(html_elem):
             )
         observed_uuids.add(uuid_value)
 
+    def check_extracted_corruption(html_elem):
+        maybe_uuid = maybe_uuid = get_extracted_elem_uuids(html_elem)
```

Comment:
Should one `maybe_uuid =` be removed here?

Reply:
:joy: -- Not sure what I was doing there. Thanks for catching that. Fixed in latest.